.gitignore

```diff
@@ -1,3 +1,4 @@
 node_modules
 build
 tmp
+**/lode-build
```

Comment:
Why not just `lode-build`?

Reply:
To ignore it inside the cli folder aswell

Reply:
Then the **/ is not required, see node modules